index.html

```diff
@@ -50,11 +50,15 @@ <h1>Todos</h1>
     </script>
 
     <script type="text/template" id="todo-item-template">
-        <input type="checkbox" class="todo-item__complete-checkbox">
-        <label class="todo-item__label">
-            {{title}}
-        </label>
-        <a class="todo-item__destroy"></a>
+        {{#unless editing}}
```

Comment:
What do you display when you aren't editing? I'd expect to see something here along the lines of 

```
{{#if editing}}
  <!-- editing html elements -->
{{else}}
  <!-- display only html elements -->
{{/if}}
```

Reply:
I did {{#unless}} {{else}} instead of {{#if}} {{else}}. I felt non-editing state was more "default" so I put it first in markup order. Do you want me to switch the order?

Reply:
Its kind of weird, you try to avoid negative checks when you can because it reads better.